src/cmd_line.rs

```diff
@@ -124,5 +131,6 @@ pub fn handle_command_line_arguments() {
         disowned: matches.is_present("disowned"),
         wsl: matches.is_present("wsl"),
         geometry: matches.value_of("geometry").map(|i| i.to_owned()),
+        frameless: !matches.is_present("frameless"),
```

Comment:
I'm confused by this. If you have a frameless commandline argument, does that set frameless to false?

If thats true, does frameless remove the frame?

Reply:
if `frameless` is `false`;  then the window **is frameless**; that is why I put a not there

Reply:
the same reason is why the default value is `true`

Reply:
That is counterintuitive to me. I expected that frameless being false would mean there is a frame. Maybe I'm confused?